Shared/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
```

Comment:
Is this copy pasted code?

Reply:
It's from the dispatcherQueue extension used here https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Uwp/Extensions/DispatcherQueueExtensions.cs since we needed an async variation of what DispatcherQueue originally had. I could adapt it to our needs tho

Reply:
A comment coming from the source was added here